teapp/src/main/java/com/techelevator/twitter/TwitterStatusListener.java

```diff
@@ -0,0 +1,72 @@
+package com.techelevator.twitter;
+
+import com.techelevator.dao.KinesisMessagePersister;
+import com.techelevator.dao.MessagePersister;
+import com.techelevator.model.PublishedMessage;
+import twitter4j.*;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+
+public class TwitterStatusListener implements StatusListener {
+
+    private static final String RESOURCE = "TWITTER";
+
+    private MessagePersister messagePersister;
+
+    public TwitterStatusListener(MessagePersister messagePersister) {
+        this.messagePersister = messagePersister;
+    }
+
+
+    @Override
+    public void onStatus(Status status) {
+        System.out.println(status.getUser().getScreenName()+"  "+status.getUser().getName() + " : " + status.getText());
+
+        String hashtags = convertHashEntities(status.getHashtagEntities());
+
+        PublishedMessage message = new PublishedMessage(
+                RESOURCE, status.getText(), status.getId(), status.getUser().getScreenName(),
+                hashtags, status.getCreatedAt().toInstant().atZone(ZoneId.systemDefault()).toLocalDate(),
+                status.getFavoriteCount(), status.getRetweetCount());
+
+        System.out.println(message.toString());
+        messagePersister.persist(message);
+    }
+
+    @Override
+    public void onDeletionNotice(StatusDeletionNotice statusDeletionNotice) {
+
+    }
+
+    @Override
+    public void onTrackLimitationNotice(int i) {
+
+    }
+
+    @Override
+    public void onScrubGeo(long l, long l1) {
+
+    }
+
+    @Override
+    public void onStallWarning(StallWarning stallWarning) {
+
```

Comment:
If we're not going to do anything with these warnings we should probably log them.